WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/ui/wizards/projectregistration/ProjectIdEnteredEndingPage.java

```diff
@@ -2,31 +2,32 @@
 
 import nl.tudelft.watchdog.logic.network.NetworkUtils;
 import nl.tudelft.watchdog.ui.wizards.IdEnteredEndingPage;
+import nl.tudelft.watchdog.ui.wizards.RegistrationWizard;
 
 /**
  * Possible finishing page in the wizard. If the user exists on the server, or
  * the server is not reachable, the user can exit here.
  */
-class ProjectIdEnteredEndingPage extends IdEnteredEndingPage {
+public class ProjectIdEnteredEndingPage extends IdEnteredEndingPage {
```

Comment:
True. Technically, in both registration wizards, there is a project registration. Therefore, maybe "UserRegistrationWizard" should be renamed to something like UserProjectRegistrationWizard.